examples/full-example/.mendelrc

```diff
@@ -17,8 +17,8 @@ variation-config:
       - partner_C
 
 route-config: # Mendel-middleware related
-  variation: /mendel/:variations/:bundle # Route pattern for variational assets during development
-  hash: /mendel/:hash/:bundle # Route pattern for variational assets during production
+  variation: /mendel/:variations/:bundle.js # Route pattern for variational assets during development
```

Comment:
I did not put the extension on purpose. Check the example app and see url of the payloads. More specifically, note the css bundle. Css bundle is not js.